go.sum

```diff
@@ -99,10 +99,8 @@ github.com/aerospike/aerospike-client-go v1.27.1-0.20170612174108-0f3b54da6bdc/g
 github.com/akavel/rsrc v0.8.0 h1:zjWn7ukO9Kc5Q62DOJCcxGpXC18RawVtYAGdz2aLlfw=
```

Comment:
It looks like this does not introduce any Go changes so I would not expect any go.sum/go.mod changes in this PR.